### DISCUSSION TITLE
Course is not moving to next step

### DISCUSSION THREAD

**Discussion by akbar401t** [skills/.github#564] [Code with Copilot]
Hi, I am stuck in step 2 in which I committed skills.js file. I refreshed the repository landing page multiple times and it did not move to next step even after 30 minutes. Anyone has solution ?

**Comment by github-cloudlabsuser-755** [comment]
Did you try looking at this issue & advice here? -  https://github.com/orgs/skills/discussions/304

FYI I had an issue with Step 3 and noticed out that if you goto the Commit History page for that repo, it should show the failed steps for that commit. 
If you click on the failure numbers it gives a summary, then you can click Details to find out more about why that step is failing -
![image](https://github.com/skills/.github/assets/167453234/7b1aab38-5551-49ee-a463-4d1936f7893e)

Knowing there are logs you can look at might help you/others in future too.

My issue was caused by the fact that the checks are case-sensitive ie "Create web server" -
![image](https://github.com/skills/.github/assets/167453234/8cca1e74-32be-4edc-bfda-a4d095ff172b)
